pkg/morph/client/netmap/add_peer.go

```diff
@@ -24,13 +24,6 @@ func (a *AddPeerPrm) SetNodeInfo(nodeInfo netmap.NodeInfo) {
 func (c *Client) AddPeer(p AddPeerPrm) error {
 	var method = addPeerMethod
```

Comment:
not needed anymore

Reply:
used in err message

Reply:
`const` then

Reply:
`addPeerMethod` is already a `const`, should be sufficient.

Reply:
thats what this thread is bout